### PR TITLE
Update UG on remove command

### PR DESCRIPTION
**Pull request by huanghao1998**
Add remove command in the UG
Update `Notes about the command format`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ SOCket is a **desktop app for NUS Software Engineering Students to manage the co
 
 **:information_source: Notes about the command format:**<br>
 
+* `INDEX` must be a positive number starting from 1 and ending with corresponding number of Person in SOCket. e.g. `1,2,...`
```
Comment (chia-yh) [inline docs/UserGuide.md:48]: Perhaps specify `integer` in particular, can refer to the lines removed in the old version

File: docs/UserGuide.md [hunk 2]
```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
...
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all persons or tags : `clear [t/TAG]`
+### Removing a person's field/attribute : `remove`
```
Comment (chia-yh) [inline docs/UserGuide.md:164]: Should we decide between field/attribute and use it consistently? Could also include the term in the glossary

File: docs/UserGuide.md [hunk 3]
```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
...
+
+Removes the specific field value based on the given input
+
+Format: `remove INDEX [n/NAME] [p/PHONE] [p/GITHUBPROFILE] [e/EMAIL] [a/ADDRESS] [l/LANGUAGE] [t/TAG]...`
```
Comment (chia-yh) [inline docs/UserGuide.md:168]: Should `[n/NAME]` be present here?

Also, perhaps consider the nested optional field as discussed, i.e., `[n/[PHONE]]` and so on, could include a reference to it in the preamble explaining its meaning.

Same issue in the command summary.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
...
+
+* Removes field value of person at the specific `INDEX`.
+* Removes all the corresponding field value in respect of the `KEYWORD`.
+  * e.g. `l/Java` will remove `Java` in person’s **Language** field.
+* Removes the entire field value when that field has no keyword provided.
+  * e.g. `t/` will remove all the tags associate to the person.
+
+### Clearing all persons or tags : `clear`
```
Comment (chia-yh) [inline docs/UserGuide.md:176]: Not sure if you'd like to include a
Format: `clear` for consistencies sake

**Review by chia-yh — COMMENTED** [review]
Some minor concerns, do take a look.

File: docs/UserGuide.md (hunk 4 above)
Comment (huanghao1998) [reply docs/UserGuide.md:176]: The Format is at line 180

File: docs/UserGuide.md (hunk 3 above)
Comment (huanghao1998) [reply docs/UserGuide.md:168]: Thanks for suggesting! I will change it in the next push

File: docs/UserGuide.md (hunk 2 above)
Comment (huanghao1998) [reply docs/UserGuide.md:164]: Yes we should do the glossary for UG!

File: docs/UserGuide.md (hunk 4 above)
Comment (chia-yh) [reply docs/UserGuide.md:176]: I see, missed that, my bad

**Pull request merged**